cmd/iceberg/main.go

```diff
@@ -53,12 +58,22 @@ Arguments:
   VALUE          value to set
 
 Options:
-  -h --help          show this helpe messages and exit
   --catalog TEXT     specify the catalog type [default: rest]
   --uri TEXT         specify the catalog URI
   --output TYPE      output type (json/text) [default: text]
   --credential TEXT  specify credentials for the catalog
-  --warehouse TEXT   specify the warehouse to use`
+  --warehouse TEXT   specify the warehouse to use
+
+Example Usages
+  iceberg list [options] [PARENT]
+  iceberg describe [options] [namespace | table] IDENTIFIER
+  iceberg (schema | spec | uuid | location) [options] TABLE_ID
+  iceberg drop [options] (namespace | table) IDENTIFIER
```

Comment:
Kept the existing set of usages because this line for example has `iceberg command [options] namespace` and  then an argument -- the `namespace` part isn't fully captured in the `iceberg command [options] [arguments]` format. Thought it'd be better to just leave these as examples.